themes/theme-ios11/pages/Favorites/subscriptions.js

```diff
@@ -1,4 +1,4 @@
-import { addFavorites } from '@shopgate/pwa-common-commerce/favorites/actions/toggleFavorites';
+import { addFavorite } from '@shopgate/pwa-common-commerce/favorites/actions/toggleFavorites';
```

Comment:
Please use the `@shopgate/engage/favorites` import here already.

Reply:
done